docs/content/en/docs/crd-ref/metrics/v1alpha3/_index.md

```diff
@@ -494,11 +495,13 @@ _Appears in:_
 
 _Appears in:_
 - [AnalysisSpec](#analysisspec)
+- [AnalysisStatus](#analysisstatus)
 
 | Field | Description |
 | --- | --- |
 | `from` _[Time](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#time-v1-meta)_ | From is the time of start for the query, this field follows RFC3339 time format |
 | `to` _[Time](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#time-v1-meta)_ | To is the time of end for the query, this field follows RFC3339 time format |
+| `recent` _[Duration](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#duration-v1-meta)_ | Recent describes a recent timeframe using a duration string. E.g. Setting this to '5m' will cause the Analysis to be performed for the last five minutes |
```

Comment:
Maybe `last` reads better? 

```yaml
timeframe:
  recent: '5m'
```

```yaml
timeframe:
  last: '5m'
```

Reply:
as agreed offline, we decided to keep `recent`